packages/expo-dev-menu/android/src/main/java/expo/modules/devmenu/DevMenuHost.kt

```diff
@@ -22,13 +23,25 @@ import java.io.InputStreamReader
  */
 class DevMenuHost(application: Application) : ReactNativeHost(application) {
 
-  override fun getPackages() = listOf(
-    MainReactPackage(null),
-    DevMenuPackage(),
-    getVendoredPackage("com.swmansion.reanimated.ReanimatedPackage"),
-    getVendoredPackage("com.swmansion.gesturehandler.react.RNGestureHandlerPackage"),
-    getVendoredPackage("com.th3rdwave.safeareacontext.SafeAreaContextPackage"),
-  )
+  override fun getPackages(): List<ReactPackage> {
+    val packages = mutableListOf<ReactPackage>(
+      MainReactPackage(null),
+      DevMenuPackage(),
+      getVendoredPackage("com.swmansion.reanimated.ReanimatedPackage"),
+      getVendoredPackage("com.swmansion.gesturehandler.react.RNGestureHandlerPackage"),
+      getVendoredPackage("com.th3rdwave.safeareacontext.SafeAreaContextPackage"),
+    )
+
+    val devLauncherPackage = Class.forName("expo.modules.devlauncher.DevLauncherDevMenuExtensionPackage")
```

Comment:
It might be good to wrap this in a try-catch as it could throw a `ClassNotFoundException` (among others)

Reply:
done here: https://github.com/expo/expo/commit/99c8d263e3fcd89e1999bb2359336d6ae28096d0